index.html

```diff
@@ -5,8 +5,70 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com"> 
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+  <link href="https://fonts.googleapis.com/css2?family=Oswald&family=Rasa&family=Spectral&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  
+<!------------------------- top ----------------------------->
+<div class="page1"> 
+<div class="oneArea"></div>
+
+  <div class="top">
+
+    <div class="top-text">
+    <h1>A Watch Makes You More Attractive</h1>
+    <p>A watch...<br>tells you a time, part of your fashion, makes you a confidence.<br>
+    Are you looking for the best watch for you?<br>
+    We would like to help you.</p>
+    </div>
+   <img src="./image/tissot-silvergold.png" >
+  </div>
+</div>
+
+
+  <!------------------------ New arrival --------------------->
+<div class="new">
+  <h2>New Arrivals</h2>
+  <div class="new-collections">
+   
+    <img src="./image/citizen-jewery.png"></li>
+    <img src="./image/mido-silvergold.png"></li>
+    <img src="./image/rado-gold.png"></li>
+    
+    <h6>See More</h6>
```

Comment:
This section has a function as a kind of like a button. So I think it's better to use <button> tag.